admin/client.py

```diff
@@ -9,6 +9,7 @@
 import tempfile
 import yaml
 
+from characteristic import attributes
```

Comment:
No action required.

FWIW, the author of characteristic now recommends people use [attrs](http://attrs.rtfd.org/)